include/seqan3/argument_parser/validators.hpp

```diff
@@ -165,10 +166,13 @@ class arithmetic_range_validator
  *
  * \details
  *
- * On construction, the validator must receive a list (vector) of valid values.
+ * On construction, the validator must receive a range or parameter pack of valid values.
  * The struct than acts as a functor, that throws a seqan3::parser_invalid_argument
  * exception whenever a given value is not in the given list.
  *
+ * \note In order to simplify the chaining of validators, all ranges of arithmetic types resolve to the validator
+ *       value type `double` and if conversion is possible to the value type std::string. Otherwise, ...
```

Comment:
Please, complete the sentence 😏 
I just wanted to make clear that the part, about what happens if both things before didn't apply, is missing.